test/integration/minishift.go

```diff
@@ -117,3 +170,47 @@ func (m *Minishift) getRoute(serviceName, nameSpace string) string {
 	cmdOut, _, _ := m.runner.RunCommand("openshift service " + serviceName + " -n" + nameSpace + " --url")
 	return strings.TrimRight(cmdOut, "\n")
 }
+
+func (m *Minishift) checkServiceRolloutForSuccess(service string, done chan bool) {
+	command := fmt.Sprintf("rollout status deploymentconfig %s --watch", service)
+
+	ocRunner := m.runner.GetOcRunner()
+	cmdOut, cmdErr, cmdExit := ocRunner.RunCommand(command)
+	commandOutputs = append(commandOutputs,
```

Comment:
I don't think append is thread safe this way. I this case you probably would have to use a mutex as well.